lib/livebook/zta/livebook_teams.ex

```diff
@@ -35,6 +35,17 @@ defmodule Livebook.ZTA.LivebookTeams do
     end
   end
 
+  @impl true
+  def logout(name, %{assigns: %{current_user: %{payload: %{"access_token" => token}}}}) do
+    team = Livebook.ZTA.get(name)
+
+    case Teams.Requests.logout_identity_provider(team, token) do
+      {:ok, _no_content} -> :ok
+      {:error, _} = error -> error
```

Comment:
According to the spec the error can also be `{:error, map()}`, so we should normalize to string?

Reply:
You're right, we only automatically normalize the map error with `Livebook.Teams` API.